_includes/newsletter-modal.html

```diff
@@ -0,0 +1,12 @@
+<div class="modal fade" id="newsletter-success" tabindex="-1" role="dialog" aria-hidden="true">
```

Comment:
Is this modal used? Or does subscribing to the newsletter redirect to a MailChimp page?

Reply:
Not yet used, but it looks like it's the modal as per design here: 
https://projects.invisionapp.com/d/main/default/#/console/17881048/379066111/preview

Reply:
But for current simplicity, since functionality is not implemented here, and passing this PR faster, I would also recommend just redirecting to our mailchimp page when clicking on `subscribe` and work on newsletter specifics at a later stage